### PR TITLE
Add MyAnimeList

### LINKED ISSUE #4602: [Icon] myanimelist.net
**Name:** MyAnimeList
**Website:** myanimelist.net
**Alexa rank:** 829 (https://www.alexa.com/siteinfo/myanimelist.net)
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by ahmubashshir**
image

**Issue:** #4602
**Alexa rank:** 829 (https://www.alexa.com/siteinfo/myanimelist.net)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Why did you pick the hex value?
    - It was default background.
  - Did you manually vectorize the logo?
    - No.
 
Closes #4602

### CODE REVIEW

File: icons/myanimelist.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M9.675 11.217c-.04 2.392.62 4.732 1.382 6.783.577-.466 1.092-.89 1.614-1.329-.583-1.205-.81-2.619-.895-3.753l2.89.027v3.591h2.217V8.667h-2.217v2.306h-2.738c.042-3.062 3.715-2.924 5.308-2.89l-.5-1.984c-3.758-.393-7.01 1.24-7.06 5.118zm9.18 5.335h4.74L24 14.418h-3.11V6.065L18.855 6zM0 6.03v10.495h2.128V9.926l1.982 3.188 1.983-3.188v6.6h2.104V6.028H6.093L4.11 9.217 2.128 6.03z"/></svg>
```
Comment (adamrusted) [inline icons/myanimelist.svg:1]: It looks like your icon (black) is a lot thinner than the actual logo underneath. Any chance you could take another look at this?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Hi @ahmubashshir, thanks for the contribution! 🎉
Just a quick change below regarding the shape of your icon.

File: icons/myanimelist.svg (hunk 1 above)
Comment (ahmubashshir) [reply icons/myanimelist.svg]: Can you check this one?
I'm new to Inkscape and didn't use any other tools at all.

File: icons/myanimelist.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M15.293 7.19c-.17 0-.307.001-.373.005-2.151.112-3.677.743-4.507 1.866-.82 1.103-.99 2.717-.498 4.714a17.62 17.62 0 001.135 3.034l1.66-1.087c.035-.033.026-.062-.066-.199-.26-.397-.498-.986-.67-1.686-.11-.438-.213-1.04-.186-1.063l2.823.012-.002 2.826 2.323.012-.021-6.314h-2.294l-.047 1.825-2.658-.014c-.036-.054-.015-.256.065-.507.267-.83.765-1.34 1.535-1.574.577-.232 3.763-.11 3.763-.11 0-.011-.116-.393-.258-.853-.246-.794-.26-.838-.335-.853a28.051 28.05 0 00-1.39-.034zm3.493.008v8.444l4.797.039.418-1.743-3.08-.074-.025-6.61zM.021 7.227l-.02 8.394h2.132v-5.214l2.022 2.513 2.023-2.454v5.155h2.09V7.227h-2.09L4.155 9.74 2.133 7.227z"/></svg>
```
Comment (adamrusted) [inline icons/myanimelist.svg:1]: There's still some differences between this and the PNG unfortunately. You may find it easier to use a combination of the basic shapes and the pen tool to try and trace the icon - as we need them to be as accurate as possible. If you could take another look at this, and get it as close to the PNG as possible, I'll also be looking at this one over the next few days - as it seems to be a bit of a troublesome path!

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/myanimelist.svg (hunk 2 above)
Comment (adamrusted) [reply icons/myanimelist.svg:1]: Just returning to this @ahmubashshir, I've generated a path manually that seems to be pretty close to the source, and at 3 decimals of precision. Feel free to compare it yourself and if it matches on your end, you're welcome to submit in the PR.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M2.218 7.248l1.871 2.548 2.004-2.548h2.208v8.416H6.175v-5.181L4.15 12.858l-1.963-2.4-.044 5.206H0V7.248h2.218zm11.739 1.73h3.32l-.538-1.73s-.972-.095-1.81-.042a11.03 11.03 0 0 1-.163.01h-.003c-1.846.112-2.938.617-3.538 1.035l-.091.07C9.7 9.455 9.659 11.113 9.659 11.113s-.117 1.694.285 2.904c.401 1.211.64 2.01 1.108 2.793l1.72-1.104s-.453-.34-.906-2.299a3.878 3.878 0 0 1-.095-.615h2.827v2.872h2.334l-.025-6.33h-2.31l-.03 1.78h-2.675c.259-2.066 2.065-2.137 2.065-2.137zm6.932 4.889V7.222L18.763 7.2v8.464h4.81L24 13.888l-3.111-.02z"/></svg>
```

**Comment by adamrusted** [comment]
> If it helps any, it looks like the font is "Revue".

Unfortunately it looks like they've _really_ done a number on this font - even converting it to paths and stretching to the same width/height as the existing text leaves some pretty major differences (black is same size, red is raw font). Happy to submit just the font version if we think that's the best 'fit'?

image

**Comment by ahmubashshir** [comment]
> > If it helps any, it looks like the font is "Revue".
> 
> Unfortunately it looks like they've _really_ done a number on this font - even converting it to paths and stretching to the same width/height as the existing text leaves some pretty major differences (black is same size, red is raw font). Happy to submit just the font version if we think that's the best 'fit'?
> 
> image

btw... how did you compare them?

**Comment by adamrusted** [comment]
@ahmubashshir In this instance I've imported the PNG into Illustrator, and am then scaling the text to try and match it.

**Review by adamrusted — COMMENTED** [review]
Thanks @ahmubashshir! By my crude comparison in Figma, there still appears to be a few tiny differences (see your version in red against the PNG) - but if another @simple-icons/maintainers could review this too - I think I'd be happy to let this through.

**Comment by alagos** [comment]
Just commenting to be notified when this be merged :)

**Comment by ericcornelissen** [comment]
@alagos, you can just press the "Subscribe" button in the sidebar below the "Reviewers" and "Labels" :wink: 

image

<sup>It says unsubscribe for me because I'm already subscribed, it should be the same for you now that you commented</sup>

**Comment by adamrusted** [comment]
In the interest of getting this merged, I've just pushed an updated path to the PR. Could another @simple-icons/maintainers who is good with PNGs take a quick look at this one?

**Review by service-paradis — APPROVED** [review]
I can see some really slight hairline differences, but looking good enough for me since we dont have any vector source!
Thank @ahmubashshir and @adamrusted 💯

**Pull request merged**